src/mock.ts

```diff
@@ -0,0 +1,65 @@
+import Dats, { Options } from './index';
+import { Socket } from './socket';
+import { URL } from 'url';
+import { debuglog, DebugLoggerFunction } from 'util';
+
+export default class DatsMock extends Dats {
```

Comment:
Do we want to export this in  the whole module?

Reply:
I think this should be a separated export so we don't pollute the main class with `ifs`

Reply:
I think this needs just a little bit of documentation to explain how to access the mock.

Reply:
Yes, I add it.